pkg/apis/enterprise/v1alpha3/indexercluster_types.go

```diff
@@ -88,6 +94,9 @@ type IndexerClusterStatus struct {
 
 	// status of each indexer cluster peer
 	Peers []IndexerClusterMemberStatus `json:"peers"`
+
+	// Splunk Smart store configuration
+	SmartStore SmartStoreSpec `json:"smartstore,ommitempty"`
```

Comment:
think you have an extra "m" here: "omitempty"

Reply:
Thanks for this nice catch Mike. I am thinking if this is the reason for the CircleCI failure. (Wondering, if it is not supposed to be caught by the compiler)